.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Issue Template
+about: Default issue template
+title: "[ISSUE]"
+labels: ''
+assignees: ''
+
+---
+
+# For New Contributors
+[How to contribute]()
+[How to setup your environment]()
```

Comment:
Is this section about providing useful information to new contributors? Both given links should be in a `CONTRIBUTING.md` and not in an issue template I think.

Reply:
Changed to one link that points to https://github.com/Bannerlord-Coop-Team/BannerlordCoop/wiki/Getting-Started-as-a-Contributor